src/components/contents/html.js

```diff
@@ -19,7 +19,7 @@ export default class HTMLView extends React.Component<*> {
       >
         <iframe
           title={`view of ${this.props.entry.path}`}
-          sandbox="allow-scripts"
+          sandbox="allow-scripts allow-popups allow-same-origin"
```

Comment:
I don't think we need `allow-same-origin` if we just want to allow people to navigate links in a generated document. Thoughts?

Reply:
Thanks @captainsafia !, I have updated the PR to remove the `allow-same-origin`